### PR TITLE
Add parfive

### PR DESCRIPTION
**Pull request by Cadair**
Adds https://github.com/Cadair/parfive

### CODE REVIEW

**Comment by Cadair** [comment]
@conda-forge/help-python I think this is ready for review :slightly_smiling_face:

File: recipes/parfive/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools_scm
+  run:
+    - python
+    - setuptools
```
Comment (marcelotrevisani) [inline recipes/parfive/meta.yaml]: Do you need ``setuptools`` to run this project?

Comment (marcelotrevisani) [reply recipes/parfive/meta.yaml]: I did a quick look at this project and didn't see ``setuptools`` as a requirement to run it. I may be wrong, but could you please double check that?

Comment (Cadair) [reply recipes/parfive/meta.yaml]: We use `pkg_resources` to get the version number [here](https://github.com/Cadair/parfive/blob/master/parfive/__init__.py#L13) as suggested by setuptools_scm, so I think that means we need setuptools?

Comment (marcelotrevisani) [reply recipes/parfive/meta.yaml]: In this case you ``setuptools`` is a dependency, thanks for double checking it! :)

**Comment by Cadair** [comment]
Thanks!

**Pull request merged**